store/sqlstore/store.go

```diff
@@ -357,6 +357,20 @@ func (ss *SqlStore) IsBinaryParamEnabled() bool {
 	return ss.isBinaryParam
 }
 
+func (ss *SqlStore) SetPgDefaultTextSearchConfig(defaultTextSearchConfig string) {
```

Comment:
No need to create a separate method for this. You can directly call `s.GetMasterX().Exec` from the test function.

Reply:
Ok I removed it. I created because I read the `pgDefaultTextSearchConfig` from the store variable every time I made the search query. I removed it now and let the query take the default_text_search_config from the database.